src/_Typography.sass

```diff
@@ -11,57 +11,57 @@ h3,
 h4,
 h5,
 h6
-	font-weight: 300
+	font-weight: $heading-font-weight
 	margin-bottom: 2.0rem
 	margin-top: 0
 
 h1
-	font-size: 4.0rem
-	letter-spacing: -.1rem
-	line-height: 1.2
+	font-size: $h1-font-size
+	letter-spacing: $heading-letter-spacing
+	line-height: $h1-line-height
 
 h2
-	font-size: 3.6rem
-	letter-spacing: -.1rem
-	line-height: 1.25
+	font-size: $h2-font-size
+	letter-spacing: $heading-letter-spacing
+	line-height: $h1-line-height
 
 h3
-	font-size: 3.0rem
-	letter-spacing: -.1rem
-	line-height: 1.3
+	font-size: $h3-font-size
+	letter-spacing: $heading-letter-spacing
+	line-height: $h1-line-height
 
 h4
-	font-size: 2.4rem
-	letter-spacing: -.08rem
-	line-height: 1.35
+	font-size: $h4-font-size
+	letter-spacing: $heading-letter-spacing / 1.25
+	line-height: $h1-line-height
 
 h5
-	font-size: 1.8rem
-	letter-spacing: -.05rem
-	line-height: 1.5
+	font-size: $h5-font-size
+	letter-spacing: $heading-letter-spacing / 2
+	line-height: $h1-line-height
 
 h6
-	font-size: 1.6rem
+	font-size: $h6-font-size
 	letter-spacing: 0
-	line-height: 1.4
+	line-height: $h1-line-height
 
 // Larger than mobile screen
-@media (min-width: 40.0rem)
+@media (min-width: $large-screen)
 
 	h1
-		font-size: 5.0rem
+		font-size: $h1-font-size * 1.25
 
 	h2
-		font-size: 4.2rem
+		font-size: $h1-font-size * 1.25
```

Comment:
@jackmcpickle Please correct font-size for headings h2-h6. They are all referring to $h1-font-size.